src/main.rs

```diff
@@ -165,6 +165,8 @@ fn main() -> Result<()> {
     let mount_injector = inject(option.clone())?;
 
     info!("waiting for signal to exit");
+    let rpc_heartbeat = "{\"method\": \"ServerHandler.Ping\", \"params\": [{}],\"id\": null}\n";
```

Comment:
A quick search gives me this repo: https://github.com/paritytech/jsonrpc . I don't know whether it's good. Maybe you can have a try :smile_cat: . It seems that it's a JSONRPC 2.0 library, so maybe we should find a JSONRPC 2.0 pkg for go :cry: .

Reply:
Now, `chaos-daemon` is the server. I don't know if it would be better to make `toda` act as a server, so that `chaos-daemon` can use some rpc calls to modify injection configuration without restarting toda in the future :smile_cat:.

Reply:
Sure thing! Now she's much prettier, sure TAL @YangKeao

Reply:
But do not work yet, WAM

Reply:
That's ok, there is no problem. It is because I run toda like
`cat ./config-examples/latency-example.json | sudo ./target/debug/toda --path /root/test --mount-only`
And tried to give rpc calls from shell. 🤦